includes/class-wp-job-manager.php

```diff
@@ -431,6 +431,19 @@ public function frontend_scripts() {
 		wp_register_script( 'wp-job-manager-job-submission', JOB_MANAGER_PLUGIN_URL . '/assets/js/job-submission.min.js', [ 'jquery' ], JOB_MANAGER_VERSION, true );
 		wp_localize_script( 'wp-job-manager-ajax-filters', 'job_manager_ajax_filters', $ajax_data );
 
+		if ( isset( $select2_args ) ) {
+			$select2_filters_args = array_merge( $select2_args, [ 'allowClear'              => true,
```

Comment:
Not sure why this isn't failing in CI, but I think this needs to be expanded to different lines (see phpcs result; phpcbf on this file should fix most of the issues). Might be worth rebasing with master at the same time as we've updated some of these rules.